<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_main_window</class>
 <widget class="QDialog" name="register_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeSoft register</string>
  </property>
  <widget class="QLabel" name="register_imglbl_img">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>620</width>
     <height>453</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/register-dark 1.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="register_widget" native="true">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>484</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QPushButton" name="register_btn_login">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>500</y>
      <width>340</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="register_txtlbl_title">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>143</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Да начнеться приключение!</string>
    </property>
   </widget>
   <widget class="QLabel" name="register_txtlbl_reg">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>552</y>
      <width>342</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Уже есть аккаунт?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="register_txtlbl_pragrah_log">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>173</y>
      <width>342</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сделайте управление вашим временем простым и увлекательным!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="register_txtlbl_password">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>391</y>
      <width>342</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="register_txtlbl_email">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>319</y>
      <width>342</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Почта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn_create">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>580</y>
      <width>342</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="register_lineEdit_password">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>414</y>
      <width>342</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>⚉⚉⚉⚉⚉⚉⚉⚉</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="register_lineEdit_email">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>339</y>
      <width>342</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>my@email.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="register_lineEdit_name">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>264</y>
      <width>342</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Ева</string>
    </property>
   </widget>
   <widget class="QLabel" name="register_txtlbl_name">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>244</y>
      <width>342</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="register_checkbox_agree">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>466</y>
      <width>342</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto-Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Я согласен с условиями использования</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="register_btn_theme">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/icons/white_and_dark.svg</normaloff>img/icons/white_and_dark.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>34</width>
      <height>34</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="register_txtlbl_logo">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>37</y>
     <width>120</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TimeSoft</string>
   </property>
  </widget>
  <widget class="QLabel" name="register_imglbl_logo">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>39</y>
     <width>36</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/main/Logo.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
